### ISSUE TITLE
Invalid requests are not handled

### ISSUE THREAD

**Issue by jennydaman** [FNNDSC/pman#135]
```
curl http://localhost:5010 --data '{}'
```

```
Request = ...
Exception in thread Thread-15:
Traceback (most recent call last):
File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
self.run()
File "/usr/local/lib/python3.8/dist-packages/pman/pman.py", line 667, in run
resultFromProcessing    = self.process(request)
File "/usr/local/lib/python3.8/dist-packages/pman/pman.py", line 2149, in process
d_request           = d_payload['payload']
KeyError: 'payload'
```

No http response, connection stays open forever (or until client/network timeout)

**Comment by jennydaman** [comment]
DOS because server listener thread is not released, making 12 bad requests occupies the entire thread pool.

**Comment by jennydaman** [comment]
Probably fixed by 18f79200d549c00461187addf4f80dd9740b0a6b
